src/components/pagination/pagination.scss

```diff
@@ -0,0 +1,265 @@
+ *    of individual items relative to this container.
+ *
+ * @see https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html
+ */
+
+.c-pagination__items {
+  display: flex; /* 1 */
+  list-style: none; /* 2 */
+  padding-left: 0; /* 2 */
+  position: relative; /* 3 */
+}
+
+/**
+ * We initialize the list's pseudo elements so they will reserve space for the
+ * "previous" and "next" affordances.
+ */
+
+.c-pagination__items::before,
+.c-pagination__items::after {
+  content: '';
+}
+
+/**
+ * Make the pseudo elements _and_ list items occupy equal amounts of space.
+ * This gives us nice, even spacing without having to know the exact number of
+ * items beforehand.
+ */
+
+.c-pagination__items::before,
+.c-pagination__items::after,
+.c-pagination__item {
+  flex: 100%;
+}
+
+/**
+ * The element that navigates to a page. Currently only intended for links, but
+ * in the future this could be extended to support `<button>` if we wanted.
+ *
+ * It's important to avoid setting `position` on this element or we'll disrupt
+ * our ability to position "previous" and "next" affordances without requiring
+ * redundant elements.
+ */
+
+.c-pagination__action {
+  align-items: center;
+  display: flex;
+  height: $action-size;
+  justify-content: center;
+  text-decoration: none;
+  user-select: none;
+}
+
+/**
+ * The current state should inherit the parent element's text color to appear
+ * just a bit muted.
+ */
+
+.c-pagination__action[aria-current] {
+  color: inherit;
+}
+
+/**
+ * When an action element has `is-previous` or `is-next` state classes applied,
+ * we absolute-position pseudo elements that provide extra "previous" and "next"
+ * affordances without requiring any additional markup.
+ */
+
+.c-pagination__action.is-previous::before,
+.c-pagination__action.is-next::after {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: sizes.$control-icon-size;
+  border: sizes.$edge-small solid colors.$gray-light;
+  border-radius: sizes.$radius-full;
+  bottom: 0;
+  content: '';
+  height: $action-size;
+  margin: auto;
+  position: absolute;
+  top: 0;
+  transition: filter motion.$speed-slow motion.$ease-out,
+    transform motion.$speed-quick motion.$ease-out;
+  width: $action-size;
+
+  .t-dark & {
+    background-color: colors.$primary-brand-dark;
+    border-color: transparent;
+  }
+}
+
+/**
+ * These affordances have similar hover and active states to buttons. We apply
+ * these styles relative to the action state (rather than the pseudo element)
+ * so hovering over a page number will _also_ trigger the animation on the
+ * affordance, which helps enforce the action being considered.
+ */
+
+.c-pagination__action.is-previous:hover::before,
+.c-pagination__action.is-next:hover::after {
+  filter: brightness(110%);
+  transform: scale(1.05);
+}
+
+.c-pagination__action.is-previous:active::before,
+.c-pagination__action.is-next:active::after {
+  filter: brightness(80%);
+  transform: scale(0.95);
+}
```

Comment:
Nit: I wonder if it would be useful to house these interaction state values as design tokens if they're shared between multiple components?

Reply:
- [x] Created design tokens for these values and referenced them across multiple components